### ISSUE TITLE
support kube_config exec

### ISSUE THREAD

**Issue by cjhodges77** [abonas/kubeclient#362]
Is there a plan to support the kube_config exec function? as in:
```console
users:
- name: user-name
  user:
    exec:
      apiVersion: client.authentication.k8s.io/v1alpha1
      command: heptio-authenticator-aws
      args:
```

**Comment by motymichaely** [comment]
@cjhodges77 I needed this as well so I just implemented it ;)

**Comment by cben** [comment]
Closed by #363 (thanks Moty again).
